l1-contracts/contracts/bridge/ntv/NativeTokenVault.sol

```diff
@@ -429,7 +429,14 @@ abstract contract NativeTokenVault is INativeTokenVault, IAssetHandler, Ownable2
             _originToken,
```

Comment:
Okay, so bridging previously not bridged tokens will not work here, since the old L1ERC20Bridge will not encode the erc20 mint data correctly, right?